docs/admin_guide.md

```diff
@@ -158,6 +158,155 @@ hoodie:trips->stats wa
 In order to limit the amount of growth of .commit files on HDFS, Hoodie archives older .commit files (with due respect to the cleaner policy) into a commits.archived file.
 This is a sequence file that contains a mapping from commitNumber => json with raw information about the commit (same that is nicely rolled up above).
 
+
+#### Compactions
+
+To get an idea of the lag between compaction and writer applications, use the below command to list down all 
+pending compactions.
+
+```
+hoodie:trips->compactions show all
+     ___________________________________________________________________
+    | Compaction Instant Time| State    | Total FileIds to be Compacted|
+    |==================================================================|
+    | 20180724223244         | REQUESTED| 35                           |
+    | 20180724215601         | INFLIGHT | 27                           |
+
+```
+
+To inspect a specific compaction plan, use
+
+```
+hoodie:trips->compaction show --instant 20180801202758
+    ____________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________
+    | Partition Path| File Id                             | Base Instant  | Data File Path                                                                                                                                                   | Total Delta Files| getMetrics                                                                                                                    |
+    |===========================================================================================================================================================================================================================================================================================================================================================================================|
+    | 2018/07/17    | 824a79ac-36d6-4d56-bd29-d182fe1a887d| 20180801184843| viewfs://ns-default/.../2018/07/17/824a79ac-36d6-4d56-bd29-d182fe1a887d_0_20180801184843.parquet | 1                | {TOTAL_LOG_FILES=1.0, TOTAL_IO_READ_MB=1230.0, TOTAL_LOG_FILES_SIZE=2.51255751E8, TOTAL_IO_WRITE_MB=991.0, TOTAL_IO_MB=2221.0}|
+
+```
+
+To manually schedule or run a compaction, use the below command. This command uses spark launcher to perform compaction
+operations. NOTE: When running these operations, make sure no other concurrent compaction scheduling is happening
+
+```
+hoodie:trips->hoodie:schemaless_jobstore_orders_rows->help compaction schedule
+              Keyword:                   compaction schedule
```

Comment:
Is this indentation right ? The one below is left indented and this one has tab indented